graftm/clusterer.py

```diff
@@ -47,7 +62,8 @@ def uncluster_annotations(self, input_annotations, reverse_pipe):
                 placed_alignment_base = placed_alignment_file.replace('_clustered.fa', '')
             output_annotations[placed_alignment_base] = {}
             for rep_read_name, rep_read_taxonomy in cluster_classifications.iteritems():
-
+                if merge_reads:
+                    rep_read_name=rep_read_name[:-2]
```

Comment:
Thanks for pointing this out. I did have self.slash_endings there to check this, but it appears I didnt put it where it was actually needed, which was silly.